scripts/qa_crowbarsetup.sh

```diff
@@ -3647,14 +3647,26 @@ function adapt_dns_for_docker
 
 function glance_image_exists
 {
-    openstack image show "$1" &>/dev/null
+    #due to https://bugs.launchpad.net/python-openstackclient/+bug/1501362
+    #images cannot be found by name over glance v2 api
+    if iscloudver 6minus; then
+        openstack --os-image-api-version 1 image show "$1" &>/dev/null
+    else
+        openstack image show "$1" &>/dev/null
+    fi
     return $?
 }
 
 function glance_image_get_id
 {
     local id=""
-    eval $(openstack image show -f shell "$1")
+    #due to https://bugs.launchpad.net/python-openstackclient/+bug/1501362
+    #images cannot be found by name over glance v2 api
+    if iscloudver 6minus; then
+        eval $(openstack --os-image-api-version 1 image show -f shell "$1" &>/dev/null)
+    else
+        eval $(openstack image show -f shell "$1" &>/dev/null)
```

Comment:
IMHO, this cannot work, because you are dropping all stderr and **stdout** , so there would never be an id= string left to eval. Probably a copy-paste-error.